internal/job/tracing.go

```diff
@@ -260,9 +261,7 @@ func Merge(maps ...map[string]any) map[string]any {
 
 	merged := make(map[string]any, fullCap)
 	for _, m := range maps {
-		for key, val := range m {
-			merged[key] = val
-		}
+		maps0.Copy(merged, m)
```

Comment:
I think it'd be better to rename the function argument than rename the "maps" import.